### ISSUE TITLE
Make the selected block outline darker 

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#12254]
As agreed on https://github.com/WordPress/gutenberg/pull/11737 the selected block outline should be darker. For non-text contrast (user interface components) the WCAG level AA contrast requirement is at least 3:1.

Right now, the outline uses a very light grey color that is barely visible for users who need it the most, especially when there are no other indicators for the selected block:

<img width="999" alt="focus after" src="https://user-images.githubusercontent.com/1682452/48358626-6f835f00-e69b-11e8-82ae-aab1fee7853c.png">

**Comment by afercia** [comment]
Note: for light themes, `$dark-opacity-200` would be OK. Not sure what color to use for dark themes.

**Comment by timwright12** [comment]
Grabbed this one via (@10up)

**Comment by afercia** [comment]
Related #12088 (duplicate).

**Comment by kjellr** [comment]
Closing now that https://github.com/WordPress/gutenberg/pull/14145 has merged. 👍
